### DISCUSSION TITLE
KIA Seltos GTX+ 2024 facelift mode

### DISCUSSION THREAD

**Discussion by anilkumarkatta204** [flowdriveai/flowpilot#70] [Q&A]
Hi All,

I have a dump question, does anyone know is there any need to install or use flowpilot on Kia Seltos GTX+ 2024 face lift model, it has alrady had some level 2 ADAS features, but curious to know if it will any of use to have this system installed on my new car?

Thanks for your answers ahead, and looking forward to hear from your all.

Thanks & Regards,
-Anil Katta

**Comment by MankaranSingh** [comment]
That's great. does it have any sort of lane keeping/correction assist ? We would need the can dumps of the car to reverse engineer the controls (non intusive). Let us know.

**Reply by anilkumarkatta204** [reply]
Sorry for the delay in response.

Yes, it has lane assistent and correction as well. if you are refering to car dumps, how can i get them?
